client/packages/cli/src/index.js

```diff
@@ -401,6 +401,7 @@ program
     '-a --app <app-id>',
     'App ID to push to. Defaults to *_INSTANT_APP_ID in .env',
   )
+  .option('-o --override', 'Override existing schema')
```

Comment:
I _think_ you don't need to do this. 

You can pass in `--yes` in the command, and it will override the existing schema by default.

Reply:
You are right!